### PR TITLE
update wcs reference file info

### PR DESCRIPTION
**Pull request by nden**
Fixed #2227

### CODE REVIEW

File: docs/jwst/assign_wcs/reference_files.rst [hunk 1]
```diff
@@ -166,8 +166,8 @@ The prism reference file has in addition the following fields:
 :angle: Angle between the front and back surface of the prosm (in degrees)
 :kcoef: K coefficients of Selmeir equation, describing the material
 :lcoef: L coeffficients describing the material
-:tcoef: Thermal coefficients describing the properties of the glass
-:tref: Reference temperature (in K)
+:tcoef: Six constants, describing the thermal behavior of the glass
+:tref: Temperature (in K), used to compute the change in temperature relative to the ref T of the glass
```
Comment (stscicrawford) [inline docs/jwst/assign_wcs/reference_files.rst]: Can you spell out ref T here?

**Comment by stscicrawford** [comment]
Thanks @nden !

Is there anything else in the ICD that should be included here?   One thing I noticed looking at a disperser file is that the gratings should include `groovedensity` and `gwa_tilty` and `gwa_tiltx` also included a unit. 

https://jwst-crds.stsci.edu/unchecked_get/references/jwst/jwst_nirspec_disperser_0035.asdf

File: docs/jwst/assign_wcs/reference_files.rst (hunk 1 above)
Comment (nden) [reply docs/jwst/assign_wcs/reference_files.rst]: done

**Comment by nden** [comment]
I added `groovedensity`. The `unit` field in `gwa_tilty/x` has no description in the IDC file but is present in the IDT reference file. It was included in the asdf file in order to replicate the IDT file but is not used. When I asked the IDT team they said the field doesn't matter.
Eventually  these should be approved by the INS team.

**Comment by stscicrawford** [comment]
One small item:  I believe the `groovedensity` is only in the grating and not in the prism.  I should finish something else up first, but after that, I'll merge this into the branch and I'll fix that if you don't have time to take care of it before I merge it.

**Comment by nden** [comment]
I'll fix it shortly ... done

**Pull request merged**